### PR TITLE
added cart page

### PR DESCRIPTION
**Pull request by tusharkandpal**
- added cart html & css
- changed products css
- added card utility since bg color was getting changed somehow. will remove later
- corrected links

### CODE REVIEW

File: components/cart-page/cart.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* ------------------------|| IMPORTS -----------------------*/
+@import url("../../utilities/main.css");
+
+/* -------------------- || CART MAIN --------------------- */
+.cart-main {
+  background-color: #f7f7f7;
```
Comment (ozairkhan2810) [inline components/cart-page/cart.css:6]: use color variable

File: components/cart-page/cart.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+/* -------------------- || CART SECTION --------------------- */
+.cart-section {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  justify-content: center;
+  gap: 3rem;
+  padding: 2rem 0;
```
Comment (ozairkhan2810) [inline components/cart-page/cart.css:20]: can combine padding property with above class to remove redundancy

File: components/product-page/products.css [hunk 3]
```diff
@@ -36,7 +36,7 @@
 
 /* ---------------------- || PRODUCTS SECTION --------------------------------*/
 .products-section {
-  background-color: #f5f5f5;
+  background-color: #f7f7f7;
```
Comment (ozairkhan2810) [inline components/product-page/products.css:39]: use color variable

**Review by ozairkhan2810 — APPROVED** [review]
suggested few changes, then can merge

File: components/cart-page/cart.css (hunk 1 above)
Comment (tusharkandpal) [reply components/cart-page/cart.css:6]: will combine all variables at last

File: components/cart-page/cart.css (hunk 2 above)
Comment (tusharkandpal) [reply components/cart-page/cart.css:20]: yeah will do.

File: components/product-page/products.css (hunk 3 above)
Comment (tusharkandpal) [reply components/product-page/products.css:39]: later on

**Review by shaksham08 — APPROVED** [review]
Looks Good 
But would say to create color variables now itself as everyone would be pointing it out and PR review can take more time then.

**Comment by tusharkandpal** [comment]
> Looks Good But would say to create color variables now itself as everyone would be pointing it out and PR review can take more time then.

just two pages are left! will do so once all main pages are in sync

**Pull request merged**